### PR TITLE
Update key features.

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
## Motivation

related #782 

## Description of the changes

- [x] Update README.
- [x] Align docs according to README.

If the readme looks good, I will update docs to follow README.

### CODE REVIEW

**Comment by crcrpar** [comment]
@keisuke-umezawa 

> If the readme looks good, I will update docs to follow README.

Do you want some approve-ish things to your README from some reviewers before applying the changes of README to docs?

**Comment by HideakiImamura** [comment]
Thanks for the great PR to improve the first impression for Optuna newbies. Key features look good to me. I'm interested in the order of key features. 

How about reversing `Pythonic search spaces` and  `Easy parallelization`? IMO, it's more understandable for users.

File: README.md [hunk 1]
```diff
@@ -27,8 +27,15 @@ Optuna can dynamically construct the search spaces for the hyperparameters.
 Optuna has modern functionalities as follows:
 
 - [Lightweight, versatile, and platform agnostic architecture](https://optuna.readthedocs.io/en/stable/tutorial/first.html)
-- [Parallel distributed optimization](https://optuna.readthedocs.io/en/stable/tutorial/distributed.html)
-- [Pruning of unpromising trials](https://optuna.readthedocs.io/en/stable/tutorial/pruning.html)
+  - Handle a wide variety of tasks, while simple installation with few requirements.
```
Comment (hvy) [inline README.md:30]: Just a nit but "while simple installation" sounded a bit unnatural. What do you think of the following?

```suggestion
  - Handle a wide variety of tasks with a simple installation that has few requirements. 
```

**Comment by keisuke-umezawa** [comment]
> @keisuke-umezawa
> 
> > If the readme looks good, I will update docs to follow README.
> 
> Do you want some approve-ish things to your README from some reviewers before applying the changes of README to docs?

Yes. Or, at least I want to have some feedback for this PR.

**Comment by keisuke-umezawa** [comment]
@HideakiImamura @hvy 
Thank you for the comment.

> As for the order, just an alternative idea in going about doing it would be to sort based on some kind of user coverage. For instance, all users are expected to: install (Lightweight, ...) / write Python (Pythonic ..) / optimize (Efficient optimization ...)

I followed this comment. But do you have any other comments, tell me that.

**Pull request merged**